adiachenko/src/helpers/context.tsx

```diff
@@ -0,0 +1,13 @@
+import React, { useContext } from "react";
+import FilmProps from "../types/types";
+
+const AppContext = React.createContext({
+    film: null,
+    sortCategory: 'all',
+    setCategory: (category: string) => null,
+    setFilm: (film: FilmProps) => null
+});
+
+const useAppContext = () => useContext(AppContext);
```

Comment:
With `AppContext` isolated together with `<AppContextProvider />` (see corresponding comment above). you could have imported that over here, thus making your custom hook implementation a bit *cleaner*.